### PR TITLE
Punctuation fixes and block padding

### PR DESCRIPTION
**Pull request by newhoggy**
* Incorporate punctuation fixes from https://github.com/input-output-hk/cardano-node/pull/1203
* And newlines above and below code blocks
* Mark JSON code blocks as such

### CODE REVIEW

File: doc/reference/shelley-genesis.md [hunk 1]
```diff
@@ -55,13 +56,16 @@ A real chain should use several genesis keys, and they should be created
 separately by the members of the federation bootstrapping the system. They
 should be created offline, kept offline and only the verification keys shared.
 
-For a demo we're going to put all the keys together in once place
+For a demo we're going to put all the keys together in once place.
+
+To start with, we will set up our template directory:
 
-To start with we will set up our template directory:
```
Comment (newhoggy) [inline doc/reference/shelley-genesis.md:60]: Punctuation fixes are above.

File: doc/reference/shelley-genesis.md [hunk 2]
```diff
@@ -55,13 +56,16 @@ A real chain should use several genesis keys, and they should be created
 separately by the members of the federation bootstrapping the system. They
 should be created offline, kept offline and only the verification keys shared.
 
-For a demo we're going to put all the keys together in once place
+For a demo we're going to put all the keys together in once place.
```
Comment (intricate) [inline doc/reference/shelley-genesis.md:59]:
```suggestion
For a demo we're going to put all the keys together in one place.
```

**Comment by intricate** [comment]
Would you be able to add the author of the last PR as a co-author on your commit or `cherry-pick` one of his commits or something?

**Comment by newhoggy** [comment]
> Would you be able to add the author of the last PR as a co-author on your commit or `cherry-pick` one of his commits or something?

I fixed up the git history.

**Pull request merged**